src/main/java/de/retest/recheck/review/ignore/matcher/ElementClassMatcher.java

```diff
@@ -15,28 +15,28 @@
 
 	public static final String CLASS_KEY = "class";
 
-	private final List<String> classValues;
+	private final List<String> classes;
 
 	public ElementClassMatcher( final Element element ) {
 		this( toClassValuesList( toClassValueString( element ) ) );
 	}
 
 	private ElementClassMatcher( final List<String> classValues ) {
```

Comment:
Please also rename the remaining "classValues" to "classes".

Reply:
Why is it important to forcibly shadow the member variable? That was IMHO the reason for this bug...